config/nodeproviders/google_config.go

```diff
@@ -1,8 +1,9 @@
 package nodeproviders
 
 type GoogleConfig struct {
-	GKENetwork    string `json:"gkeNetwork,omitempty" yaml:"gkeNetwork,omitempty"`
-	GKEProjectID  string `json:"gkeProjectID,omitempty" yaml:"gkeProjectID,omitempty"`
-	GKESubnetwork string `json:"gkeSubnetwork,omitempty" yaml:"gkeSubnetwork,omitempty"`
-	Region        string `json:"region,omitempty" yaml:"region,omitempty"`
+	AuthEncodedJSON string `json:"authEncodedJson" yaml:"authEncodedJson"`
+	GKENetwork      string `json:"gkeNetwork,omitempty" yaml:"gkeNetwork,omitempty"`
```

Comment:
nit; you rm'd `azure` from parameters with structs that contain name `azure`, why not do the same here?

Reply:
Fair point, I'll update that.

Reply:
Resolved